pkg/pillar/types/global.go

```diff
@@ -114,6 +117,165 @@ func (gs *GlobalStatus) UpdateItemValuesFromGlobalConfig(gc GlobalConfig) {
 // Agents subscribe to this info to get at least the log levels
 // A value of zero means we should use the default
 // All times are in seconds.
+type ConfigItemType uint8
+
+const (
+	CONFIGITEM_TYPE_INT ConfigItemType = iota + 1
```

Comment:
You can do:

CONFIGITEM_TYPE_INT ItemType = iota + 1

You need to do this only for the first iitem.. If you define like this, you don't need to typecast when you assign the value to a variable of type ItemType